composites/Plugin/SnippetEditor/components/SnippetEditor.js

```diff
@@ -385,24 +385,31 @@ class SnippetEditor extends React.Component {
 	 * @returns {Object} The data for the preview.
 	 */
 	mapDataToMeasurements( originalData ) {
-		const { baseUrl, mapDataToPreview } = this.props;
+		const { baseUrl, mapEditorDataToPreview } = this.props;
 
 		let description = this.processReplacementVariables( originalData.description );
 
 		// Strip multiple spaces and spaces at the beginning and end.
 		description = stripSpaces( description );
 
-		const mappedData = {
+		const shortenedBaseUrl = baseUrl.replace( /http:\/\//ig, "" );
```

Comment:
A minor point: The base url should only ever start with `http://` and never contain more in the middle somewhere. We could optimize the regex to:
```
baseUrl.replace( /^http:\/\//i, "" );
```